src/CookieThrower.cpp

```diff
@@ -12,6 +13,59 @@ CookieThrower::CookieThrower() {
     cookieShape->loadMesh(RESOURCE_DIR + "sphere.obj");
     cookieShape->resize();
     cookieShape->init();
+
+    //
+    ShaderManager& shaderManager = ShaderManager::instance();
+    std::shared_ptr<Program> progPhong = shaderManager.getShaderProgram("Phong");
+
+    static std::shared_ptr<Shape> shape = std::make_shared<Shape>();
+    static bool hasLoaded = false;
+
+    if (!hasLoaded) {
+        shape->loadMesh(RESOURCE_DIR + "sphere.obj");
+        shape->resize();
+        shape->init();
+        hasLoaded = true;
+    }
+
+    BunnyRenderComponent* bunnyRenderComp = new BunnyRenderComponent(shape, progPhong, brass);
```

Comment:
`progPhong` won't allow this program to build. Reed changed the Render components to take a string now. See #33